### PR TITLE
change voice-wave avtar title

### PR DESCRIPTION
**Pull request by mritunjayz**
#2193

### CODE REVIEW

**Comment by mritunjayz** [comment]
#2192

File: web/locales/en/messages.ftl [hunk 1]
```diff
@@ -322,7 +322,7 @@ skip-submission-note = Note: You will still need to select between Speak or List
 off = Off
 on = On
 add-avatar-title = Add an avatar to your profile
-add-avatar-clip = Add an avatar to your profile and say “hello!”
+add-avatar-clip = Give your avatar a voice
```
Comment (Gregoor) [inline web/locales/en/messages.ftl]: can we change the id of this string? that way communities in other languages will be prompted to add a new string

Comment (mritunjayz) [reply web/locales/en/messages.ftl]: Yeah, right

**Comment by Gregoor** [comment]
grazie :) 🚀

**Pull request merged**